### PR TITLE
fixing repo url and tutorial text

### PR DESCRIPTION
**Pull request by cekvenich**
We updated our website and tutorial.

Also, we broke the repo url. This should fix the tutorial and the urls.

### CODE REVIEW

File: content/projects/metabake.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
 ---
 title: Metabake
-repo: metabake/mbake-CLI-for-Metabake
-homepage: http://www.metabake.net
+repo: metabake/MetaBake-Examples
+homepage: http://mBake.org
```
Comment (erquhart) [inline content/projects/metabake.md]: This results in some sort of text file being downloaded, no site.

File: content/projects/metabake.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
 ---
 title: Metabake
-repo: metabake/mbake-CLI-for-Metabake
-homepage: http://www.metabake.net
+repo: metabake/MetaBake-Examples
```
Comment (erquhart) [inline content/projects/metabake.md]: The repo should be the repo of the static site generator itself.

File: content/projects/metabake.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
 templates: Pug and Markdown
-description: Low-code productivity for programmers via generators for Pug, Markdown and much more; including dynamic data binding.
+description: MetaBake is the extensible open source low-code productivity tool for programmers; including dynamic apps and data binding.
 ---
 
 
-#### Metabake.org is a low-code productivity for application developers via static generation; with Pug, Markdown and more; including dynamic data binding. *Some developers implement applications faster than others.*
+## MetaBake is the extensible open source low-code productivity tool for programmers; including dynamic apps and data binding.
 
+#### MetaBake is the extensible open source low-code productivity tool for programmers, via static generation; with Pug, Markdown and more; including dynamic apps and data binding. *'All. My. Friends. Know a low-coder.'*
```
Comment (erquhart) [inline content/projects/metabake.md]: This seems to be a near duplicate of the line above it.

File: content/projects/metabake.md [hunk 4]
```diff
@@ -44,15 +49,18 @@ mbake .
...
-Metabake supports CSS classes in Markdown, plus, because it uses Pug - it can also do any HTML layout. But Metabake is not static only - it fully supports and has examples and docs for dynamic apps.
+There are many example apps, and shipped templates include include an CMS module, a watcher module, SPA, Blog, Website, Slides, Dashboard, CRUD, PWA, Electron, Hybrid mobile apps, Cloud v2.0 via AWS|FireStore, RIOTjs, Ads and more. 
+
+Primary focus is high development productivity (via "low-code") and being easy to learn. It is also fully flexible to build any WebApp in any directory tree structure you like an use any CSS/SASS framework you like. Of course it is server-less, and it uses AWS S3, Google FireStore, Digital Ocean Spaces, etc.
+
+MetaBake supports CSS classes in Markdown, plus, because it uses Pug - it can also do any HTML layout. But MetaBake is not static only - it fully supports and has examples, shipped apps, and docs for dynamic and even mobile apps.
+
+[mBake.org](http://mBake.org)
```
Comment (erquhart) [inline content/projects/metabake.md]: Needs update.

File: content/projects/metabake.md (hunk 1 above)
Comment (cekvenich) [reply content/projects/metabake.md]: I had a typo in DNS. It has to DNS cache expire ~ 30 minutes. In any case I'll keep the site up.

File: content/projects/metabake.md (hunk 2 above)
Comment (cekvenich) [reply content/projects/metabake.md]: OK, fixed. I assume I added corrected code to the PR correctly.

File: content/projects/metabake.md (hunk 3 above)
Comment (cekvenich) [reply content/projects/metabake.md]: fixed in add to PR if I did it correctly.

File: content/projects/metabake.md (hunk 4 above)
Comment (cekvenich) [reply content/projects/metabake.md]: fixed in add to PR if I did it correctly.

**Pull request merged**